### PR TITLE
add art19 revision number and publishing instructions

### PR DESCRIPTION
**Pull request by tatthurs**
Add instructions for gem publishing. I added a revision number to the version and instructions for publishing.

Here is the published gem: https://github.com/orgs/art19/packages/rubygems/package/jsonapi.rb

### CODE REVIEW

File: spec/errors_spec.rb [hunk 1]
```diff
@@ -175,7 +175,10 @@
       let(:user_id) { nil }
       let(:note_id) { 'tada' }
 
-      it do
+      # TODO: This spec passes locally but fails under CI. Get this spec
```
Comment (rortian) [inline spec/errors_spec.rb:178]: "works on my box" smh

**Review by rortian — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -394,6 +389,18 @@ This project is intended to be a safe, welcoming space for collaboration, and
 contributors are expected to adhere to the
 [Contributor Covenant](http://contributor-covenant.org) code of conduct.
 
+### ART19 Gem Publishing
+
+Releases are manual, performed locally on a developer's machine. Gems are published to Github Packages. A comprehensive outline of this process can be found here: https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-rubygems-registry.
+
+1. Increment the `ART19_REVISION` in [lib/jsonapi/version.rb#L5](https://github.com/art19/jsonapi.rb/blob/master/lib/jsonapi/version.rb#L4)
+
+2. Publish the gem:
```
Comment (lexahu) [inline README.md:398]: I think we may need to add to this -- 

I remember having to do 

bundle exec rake build

mv PACKAGE.VERSION  pkg/

bundle exec rake publish

unless this is not what worked for you when publishing and I am just doing it wrong

**Review by lexahu — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (tatthurs) [reply README.md:398]: bundle exec rake publish _should_ take care of everything -- I've been standardizing the same process across the jsonapi.rb, jsonapi-serializer, cequel and cassandra_driver gems. But, if something is off here we can correct the next time we publish.

**Pull request merged**